usaspending_api/search/tests/integration/spending_by_category/test_spending_by_funding_agency.py

```diff
@@ -79,8 +60,8 @@ def test_correct_response(client, monkeypatch, elasticsearch_transaction_index,
         "limit": 10,
         "page_metadata": {"page": 1, "next": None, "previous": None, "hasNext": False, "hasPrevious": False},
         "results": [
-            {"amount": 10.0, "name": "Funding Toptier Agency 4", "code": "TA4", "id": 1004},
```

Comment:
Similar as the above comment. Need to double check the fixture changes, but the difference in the TA2 agency having an ID of 1006 meant that it successfully grabbed the ID of a different agency that had the toptier flag.

Reply:
The flipping of order was a quirk (I assume of splitting the fixtures), but these similarly were defined in lines 7 and 26 of the fixtures, and have slightly different IDs to better avoid collisions. Unless there's an error in the helper method of that file, anything with "Toptier" in it should ALWAYS be toptier, since nothing is named by hand anymore.